src/styles/blocks/header.scss

```diff
@@ -0,0 +1,59 @@
+.logo {
+  position: absolute;
+  width: 40px;
+  height: 40px;
+  left: 50px;
+  top: 10px;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  align-items: center;
+  box-shadow: 0 2px 4px rgba (0, 0, 0, 0.05);
+  padding: 0 40px 0 50px;
+}
+
+.nav-list {
```

Comment:
Check the 8th point from the [checklist](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)

Reply:
⚠️ didn't fixed